src/homework5/homework5.sql

```diff
@@ -0,0 +1,113 @@
+USE baseball;
+
+DROP TABLE IF EXISTS away_team_batter_stats;
+
+CREATE TABLE away_team_batter_stats AS
+SELECT
+    game_id,
+    team_id,
+    atBat AS AB,
+    Hit AS H,
+    Single AS singles, -- using plural because 'Double' is a key word
+    team_batting_counts.Double AS doubles, -- have to use team_batting_counts because 'Double' is a key word
+    Triple AS triples,
+    Home_Run AS HR,
+    Hit_By_Pitch AS HBP,
+    Sac_Fly AS SF,
+    Walk AS BB
+FROM team_batting_counts
+WHERE homeTeam = 0;
+
+CREATE INDEX away_team_batter_stats_game_id_idx ON away_team_batter_stats (game_id);
+
+DROP TABLE IF EXISTS home_team_batter_stats;
+
+CREATE TABLE home_team_batter_stats AS
+SELECT
+    game_id,
+    team_id,
+    atBat AS AB,
+    Hit AS H,
+    Single AS singles, -- using plural because 'Double' is a key word
+    team_batting_counts.Double AS doubles, -- have to use team_batting_counts because 'Double' is a key word
+    Triple AS triples,
+    Home_Run AS HR,
+    Hit_By_Pitch AS HBP,
+    Sac_Fly AS SF,
+    Walk AS BB
+FROM team_batting_counts
+WHERE homeTeam = 1;
+
+CREATE INDEX home_team_batter_stats_game_id_idx ON home_team_batter_stats (game_id);
+
+DROP TABLE IF EXISTS away_starting_pitcher_stats;
+
+CREATE TABLE away_starting_pitcher_stats AS
+SELECT
+    game_id,
+    team_id,
+    outsPlayed / 3 AS IP,
+    Walk AS BB,
+    Hit AS H,
+    atBat AS AB,
+    Strikeout AS K
+FROM pitcher_counts
+WHERE startingPitcher = 1
+AND homeTeam = 0;
+
+CREATE INDEX away_starting_pitcher_stats_game_id_idx ON away_starting_pitcher_stats (game_id);
+
+DROP TABLE IF EXISTS home_starting_pitcher_stats;
+
+CREATE TABLE home_starting_pitcher_stats AS
+SELECT
+    game_id,
+    team_id,
+    outsPlayed / 3 AS IP,
+    Walk AS BB,
+    Hit AS H,
+    atBat AS AB,
+    Strikeout AS K
+FROM pitcher_counts
```

Comment:
maybe you could try to use team_pitching_counts table instead of pitcher_counts, I remember the professor recommended that one more than the pitcher_counts.